sos/plugins/npm.py

```diff
@@ -44,13 +46,49 @@ def _get_npm_output(self, cmd, filename, working_directory=None):
             runat=working_directory
         )
 
+    def _find_modules_in_npm_cache(self):
+        """
+        since format of `~/.npm` is changing with newer npm releases, let's do
+        the easiest thing and don't predict nor expect anything, the structure
+        is suppose to be:
```

Comment:
I've tried npm v1, v2, v3 and latest upstream v4 and the structure as outlined here hasn't changed.
The only thing that changed recently is where the `.cache.json`s are stored - previously they were in `.npm/name/` while now they are in separate `.npm/registry.npmjs.org/name/`. Is that what you meant in the comment Tomas ?